modules/asset-registry/Cargo.toml

```diff
@@ -9,19 +9,19 @@ log = { version = "0.4.17", default-features = false }
 serde = { version = "1.0.136", optional = true }
```

Comment:
The main question I have: Can we use orml's implementation of this pallet?

Reply:
It adds an overhead for cross-chain messaging between the chains, which would not be useful for standalone chain. But since it is just our testing environment, I think that would be okay. Gonna use ORML one instead of our own version.